core/src/main/scala/kafka/cluster/Broker.scala

```diff
@@ -133,6 +134,15 @@ case class Broker(id: Int, endPoints: Map[SecurityProtocol, EndPoint]) {
     endPoints.contains(protocolType)
   }
 
+  def getNode(protocolType: SecurityProtocol): Node = {
+    val endpoint = endPoints.get(protocolType)
```

Comment:
I guess you followed the same style as `getBrokerEndPoint`. Feel free to make that better too. :)